src/components/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,125 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+const generateBoundClassNames = bindClassNames('RadioButton');
+const {
+	bool,
+	func,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
+ *
+ * This is a toggle -- a component that is in one of two particular states at
+ * any given moment in time -- that features a custom visualization of the
+ * native radio button control to reflect its current state.
+ *
+ * The radio button is different from a standard toggle in that when it is in
+ * the selected state user events do not cause it to toggle to the unselected
+ * state so the `onSelect` function is called only when `isSelected` is false.
+ *
+ * It uses a hidden native radio button control under the hood but leverages
+ * other HTML elements to visualize its state.
+ */
+const RadioButton = React.createClass({
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates whether the component should appear and act disabled by
+		 * having a "greyed out" palette and ignoring user interactions.
+		 */
+		isDisabled: bool,
+
+		/**
+		 * Indicates that the component is in the "selected" state when true
+		 * and in the "unselected" state when false.
+		 */
+		isSelected: bool,
+
+		/**
+		 * Called when the user clicks on the component or when they press the
```

Comment:
I do wrap at 80 characters. Some of my co-workers get annoyed when pairing with me on my screen because I have Sublime's demarcation line at 80 characters all the way down the window. :)